tests/unit/test_yaml.py

```diff
@@ -9,3 +13,22 @@ def test_check_yaml_imports():
     assert yaml_import.yaml is not None
     assert yaml_import.Dumper is not None
     assert yaml_import.Loader is not None
+
+
+def test_no_anchor_alias():
```

Comment:
Looks like it's just a duplicated test that could be a part of a parametrized implementation.

Reply:
done